arp-cam.c

```diff
@@ -0,0 +1,538 @@
+/******************************************************************************
+ *
+ *   Copyright (C) 2017-2018 Pelagicore AB. All rights reserved.
```

Comment:
We should take the opportunity to change any copyrights referring to Pelagicore. Should be "Luxoft Sweden AB".